src/Symfony/Component/Translation/DependencyInjection/DataCollectorTranslatorPass.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\Translation\TranslatorBagInterface;
+
+/**
+ * @author Christian Flothmann <christian.flothmann@sensiolabs.de>
+ */
+class DataCollectorTranslatorPass implements CompilerPassInterface
+{
+    /**
+     * @return void
+     */
```

Comment:
As we target 6.4, what about using native return type here (and in other moves CompilerPass)?

Reply:
I think we can't because the current classes extend the new ones unless I just copy-paste the code?

Reply:
I'd be fine duplicating the code.